src/clr_bundle_add.c

```diff
@@ -45,6 +45,8 @@ static void print_help(const char *name)
 	printf("Help Options:\n");
 	printf("   -h, --help              Show help options\n");
 	printf("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
+	printf("   -c, --contenturl=[URL]  RFC-3986 encoded url for content file downloads\n");
+	printf("   -v, --versionurl=[URL]  RFC-3986 encoded url for version string download\n");
```

Comment:
My plan is to use "levels of quality" with different versionurl, like latest build, QA-approved (manually tested) and milestones. They would only provide different numbers of the same content-stream/url.

Then how to do that with your suggested approach?

I'd expect an overlay in /etc, where /usr would default to something (i.e.: latest build) and user can set /etc to another (i.e.: qa-approved). We'd cook images with one or another pre-set, that is, the image would come with a state in /etc. this is not optimal, so suggestions are welcome.